spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,49 @@
+# Required files for testing
+require 'spec_helper'         # Load RSpec configuration and helpers
+require './lib/facility_factory'  # Load the FacilityFactory class
+require './lib/facility'          # Load the Facility class
+
+# Describe the test suite for the FacilityFactory class
+RSpec.describe FacilityFactory do
```

Comment:
These are great tests for the colorado "style" of data - very thorough!  I also like that you are using a sample of data, instead of the entire dataset. Just keep an eye out that different states have differently formatted data.